api/v01/vrp.rb

```diff
@@ -174,7 +178,8 @@ class Vrp < APIBase
                   status: job&.status&.to_sym || :completed, # :queued, :working, :completed, :failed
                   avancement: job&.message,
                   graph: solution[:graph]
-                }
+                },
+                geojsons: solution[:result]&.first&[:geojsons] || nil
```

Comment:
It is not written like this because there are several checks to make before calling OutputHelper::Result.generate_geometry() but we do call this every time we call get (from the time job is completed).